### DISCUSSION TITLE
chatwoot does not perform recompilation

### DISCUSSION THREAD

**Discussion by cwmkthubconnect** [chatwoot/chatwoot#8212] [Self hosted]
### Describe the bug

In this video we added a word along the way

app/javascript/dashboard/i18n/locale/pt_BR/resetPassword.json

and we tried to rebuild chatwoot, and it doesn't even rebuild

Just for testing purposes change this
any modification of the result and even

https://github.com/chatwoot/chatwoot/assets/91642837/16bdb8e6-4ff5-4715-a0b9-38d24daeb181

Version Chatwoot 3.2.0

### To Reproduce

app/javascript/dashboard/i18n/locale/pt_BR/resetPassword.json

Add "Chat" word

edit a file and try to rebuild

### Expected behavior

compilation

### Environment

Linux VM

### Cloud Provider

Other [please specify in the description]

### Platform

Browser

### Operating system

Ubuntu 22.04

**Comment by jtayrone** [comment]
Is anyone else going through this? In version v3.2.0, when we modify any chatwoot file, especially the translation file, the compilation does not work.
